project_recalculate/models/project_task.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, fields, api, _
+from openerp.exceptions import Warning, ValidationError
+from datetime import datetime
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    from_days = fields.Integer(
+        string='From days',
+        help='Anticipation days from date begin or date end', default=0)
+    estimated_days = fields.Integer(
```

Comment:
So, `planned_hours` is the task effort and `estimated_days` is usually known as the Task Duration.
What are the `from_days` then? A lag time after/befor the Task?